src/data-types-lib/index.js

```diff
@@ -31,7 +31,7 @@ export const
     }
 
     return {
-      format(value) {
+      format(value, { i18n } = {}) {
```

Comment:
Default value does not make any sense because
1. **i18n** is queried with **formatDecimalNumer()**.
2. **i18n** is required according to **src/crudeditor-lib/index.js**

Change arguments from `(value, { i18n } = {})` to `({ value, i18n })`